<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DateTimeChangedDialog</class>
 <widget class="QDialog" name="DateTimeChangedDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Apply the time difference to:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_photo">
       <property name="text">
        <string>Photo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cam">
       <property name="text">
        <string>All in camera</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_fol">
       <property name="text">
        <string>All in folder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_all">
       <property name="text">
        <string>All</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
